omnigibson/utils/asset_conversion_utils.py

```diff
@@ -0,0 +1,2481 @@
+import io
```

Comment:
I felt like I have reviewed this file before (maybe from the pipeline repo?). I roughly reviewed this and trusted that you guys tested this thoroughly. One thing I noticed is that many functions in this file seem deprecated and not used, or maybe I am missing something?

Reply:
Yeah I think this is just roughly copied from our internal b1k pipeline repo. I'll clean up the unused stuff